src/libsync/capabilities.cpp

```diff
@@ -148,4 +149,14 @@ QString Capabilities::invalidFilenameRegex() const
 {
     return _capabilities["dav"].toMap()["invalidFilenameRegex"].toString();
 }
+
+bool Capabilities::uploadConflictFiles() const
+{
+    static auto envIsSet = !qEnvironmentVariableIsEmpty("OWNCLOUD_UPLOAD_CONFLICT_FILES");
+    static int envValue = qEnvironmentVariableIntValue("OWNCLOUD_UPLOAD_CONFLICT_FILES");
+    if (envIsSet)
+        return envValue != 0;
+
+    return _capabilities["uploadConflictFiles"].toBool();
```

Comment:
Does that really make sense in the capabilities?
We were talking about a config endpoint. @PVince81, what do you say?

Reply:
It's in capabilities for convenience. It's not a capability and I'm fine with having it available through a config endpoint. This isn't a new issue, many things in capabilities are actually options.